test/network_autoconfiguration_test.rb

```diff
@@ -221,6 +221,11 @@ def probe_netcard_factory(num)
       let(:interfaces) { Y2Network::InterfacesCollection.new([eth0]) }
       let(:proposal) { true }
 
+      after(:each) do
+        # some methods might have sideeffects - clen them :-/
+        Yast::NetworkInterfaces.Devices.reject! { |k, _| k == "br" }
```

Comment:
and do we still need it? I  though that after my adaptation it is no longer needed as we do not read NetworkInterfaces.